### PR TITLE
install browserify with babel

### PR DESCRIPTION
**Pull request by pastak**
Build js files with browserify

exec command `npm run build`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,7 +15,8 @@
   },
   "scripts": {
     "test": "standard \"./**/*.js\"",
-    "pack": "crx pack ./ --zip-output ./build/chrome.zip -o ./build/chrome.crx",
+    "build": "mkdir -p dist && cp -r src/statics/* dist && browserify src/normal/index.js -t babelify -o dist/main.js && browserify src/common/content/*.js -t babelify -o dist/content.js",
```
Comment (uiureo) [inline package.json]: you can write the transform option in `browserify.transform` field

```
"browserify": {
  "transform": [ "babelify" ]
}
```
https://github.com/substack/browserify-handbook#browserifytransform-field

File: package.json [hunk 2]
```diff
@@ -30,7 +36,7 @@
       "KeyboardEvent"
     ],
     "ignore": [
-      "libs/jquery-1.10.2.min.js"
+      "gulpfile.js"
```
Comment (uiureo) [inline package.json]: this is unneccesary now

Comment (pastak) [inline package.json]: :eyes:

**Comment by uiureo** [comment]
LGTM

**Pull request merged**